loony_dev/agents/coding.py

```
@@ -64,6 +65,234 @@ def execute(self, task: Task) -> TaskResult:
+                stderr=subprocess.DEVNULL,
+            ).decode().strip()
+        except Exception as exc:
+            logger.warning("Could not determine repo name for PR creation: %s", exc)
+            repo_name = None
+
+        cmd = ["gh", "pr", "create", "--assignee", "@me", "--title", title, "--body", body, "--head", branch]
+        if repo_name:
+            cmd += ["-R", repo_name]
+
+        try:
+            result = subprocess.run(cmd, cwd=self.work_dir, capture_output=True, text=True, check=True)
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)